pkgs/tools/text/validator-nu/default.nix

```diff
@@ -10,14 +10,14 @@
 
 let
   pname = "validator-nu";
-  version = "22.9.29";
+  version = "2023-12-18";
 
   src = fetchFromGitHub {
     owner = "validator";
     repo = "validator";
-    rev = version;
+    rev = "73b1d25b011dab2e12cdabdefeebfe3c1a2af3cd";
```

Comment:
This isn’t the current commit in the repository, albeit for an interesting reason: the GitHub web interface displays the commits in the wrong order, because the committer dates recorded in the commits are not monotonic. It’s usual to see non-monotonic author dates due to rebasing, but non-monotonic committer dates are very strange and can cause Git to [miscompute](https://git-scm.com/docs/commit-graph) some topological relationships.

```
$ git log --graph --format='%h %cs%d %s' 23.4.11^..origin
* c3a401fe 2023-12-18 (origin/main, origin/HEAD) Update htmlparser
* 73b1d25b 2024-01-09 Add links to 3rd-party tool
* f1c50f65 2024-01-04 Update docs
* 18b9f741 2023-11-27 Update css-validator
* 92bd53a2 2023-11-09 Update docs
* 8c69d1ae 2023-09-23 Add --allowed-address-type option (#1638)
* ed62b92a 2023-08-24 Update htmlparser
* 65dc6939 2023-07-08 allow doc-glossary on aside element
* b118bf60 2023-07-03 Bump htmlparser from `8c718d3` to `9448adc`
* a6834e57 2023-07-01 Docker: Stop using distroless image
* 12656202 2023-06-22 CI: Use explicit python version
* f566ba14 2023-06-16 CI: (re)Remove Java 21-ea from the build matrix
* bcc533fe 2023-06-14 Docker: Fix syntax error
* 4dc3fb4a 2023-06-11 CI: Try adding back Java 21-ea to the build matrix
* 3ec74091 2023-06-10 Docker: Update unzip version to 6.0-28
* e39414d5 2023-06-08 Revert "Fix conformance error in site/nu-about.html"
* 7bb55dae 2023-06-07 Update CHANGELOG.md
* 83fc336e 2023-06-05 Report errors as expected for invalid named character references
* c69bc5b8 2023-05-31 Revert "Re-fix error reporting for invalid named character references"
* 07334b8f 2023-05-30 Fix conformance error in site/nu-about.html
* 093a9a09 2023-05-29 Re-fix error reporting for invalid named character references
* d21ecbf3 2023-05-01 CI: Disable Java 21-ea build for now (it’s failing on all platforms)
* 470d919b 2023-04-10 (tag: 23.4.11, tag: 23.4.10.1) Enable filtering out “Trailing slash” message without non-zero exit
```

Reply:
I changed the commit, thanks very much for catching that. It's really too bad GitHub shows commits in the wrong order.